baggers.js

```diff
@@ -1,5 +1,90 @@
 var data = {
   speakers: [
+    {
+      since: '2015-03-10',
+      name: 'Xavier Pigeon',
+      bio:
+        'Expert Méthode & Qualité (Freelance) | auteur de GearsOfTesting.org | développeur de TestAsYouThink.org',
+      picture:
+        'https://gravatar.com/userimage/29545108/d3a9e2ed3d93ef53aff440e053e0faed.jpeg',
+      websites: [
+        {
+          name: 'Website',
+          url: 'https://xavierpigeon.com/',
+        },
+        {
+          name: 'Chrysocode',
```

Comment:
Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

Reply:
Done.